src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManagerSpec.groovy

```diff
@@ -56,11 +58,11 @@ class LSFJobManagerSpec extends Specification {
       "RUNTIMELIMIT":"00:10:00",
       "SUB_CWD":"$HOME",
       "PEND_REASON":"",
-      "EXEC_CWD":"\\/home\\/otptest",
+      "EXEC_CWD":"\\/some\\/test",
```

Comment:
These changes are all in #135. You could consider to rebase (or merge) this on #135.